streams/src/main/java/org/apache/kafka/streams/kstream/JoinWindows.java

```diff
@@ -24,6 +24,24 @@
 
 /**
  * The window specifications used for joins.
+ * <p>
+ * A {@link JoinWindows} instance defines a join over two stream on the same key and a maximum time difference.
+ * The time difference can either be positive, negative, or both. In SQL-style you would express this join as
```

Comment:
"The time... or both." This may be confusing for developers as it seems they can set the window size parameter as negative values. I recommend removing this sentence as the follow-up example is sufficient.